UM/PluginRegistry.py

```diff
@@ -499,9 +501,6 @@ def uninstallPlugin(self, plugin_id: str) -> Dict[str, str]:
         result = {"status": "error", "message": "", "id": plugin_id}
         success_message = i18n_catalog.i18nc("@info:status", "The plugin has been removed.\nPlease restart {0} to finish uninstall.", self._application.getApplicationName())
 
-        if plugin_id not in self._plugins_installed:
-            return result
```

Comment:
now that we don't return the result on "error" anymore it doesn't make alot of sense to define the result object above; i would remove it and only define it at the return point of the function

Reply:
https://github.com/Ultimaker/Uranium/commit/188cf82c996f6469e8cfd29278cb194937fe2f7e